src/js/delete_wordles.js

```diff
@@ -0,0 +1,53 @@
+"use strict";
+
+/* global globalOptions, inverted_wordles, openDialog, closeDialog */
+
+// Bind events for delete buttons
+inverted_wordles.bindDeleteEvents = function (containerElm) {
+    const delButtons = containerElm.querySelectorAll(globalOptions.selectors.deleteButton);
+    for (let i = 0; i < delButtons.length; i++) {
+        const currentButton = delButtons[i];
+        // Open the delete confirmation dialog
+        currentButton.addEventListener("click", evt => {
+            openDialog(globalOptions.deleteDialogId, evt.target.id, globalOptions.deleteCancelId);
+            const deleteDialog = document.getElementById(globalOptions.deleteDialogId);
+            // set the aria-controls attribute to the id of the wordle row that will be deleted
+            const uniqueIdSuffix = evt.target.id.substring(globalOptions.length);
```

Comment:
More helpful to package this computation as a function which computes a suffix given a target id, as well as a the matching function which sticks them back together again on line 16 and in the other places where this relationship between ids is relied upon